modules/rosa-sts-creating-a-cluster-with-customizations-cli.adoc

```diff
@@ -42,9 +42,56 @@ To successfully install ROSA clusters, use latest version of the ROSA CLI.
 +
 [source,terminal]
 ----
-$ rosa create account-roles --mode manual <1>
+$ rosa create account-roles --interactive --mode manual <1><2>
 ----
-<1> `manual` mode generates the `aws` CLI commands and JSON files needed to create the account-wide roles and policies. After review, you must run the commands manually to create the resources.
+<1> `interactive` mode enables you to specify configuration options at the interactive prompts. For more information, see _Interactive cluster creation mode reference_.
+<2> `manual` mode generates the `aws` CLI commands and JSON files needed to create the account-wide roles and policies. After review, you must run the commands manually to create the resources.
++
+--
+.Example output
+[source,terminal]
+----
+I: Logged in as '<red_hat_username>' on 'https://api.openshift.com'
+I: Validating AWS credentials...
+I: AWS credentials are valid!
+I: Validating AWS quota...
+I: AWS quota ok. If cluster installation fails, validate actual AWS resource usage against https://docs.openshift.com/rosa/rosa_getting_started/rosa-required-aws-service-quotas.html
+I: Verifying whether OpenShift command-line tool is available...
+I: Current OpenShift Client Version: 4.11.6
+I: Creating account roles
+? Role prefix: ManagedOpenShift <1>
+? Permissions boundary ARN (optional): <2>
+? Path (optional): [? for help] <3>
+? Role creation mode: auto <4>
+I: Creating roles using 'arn:aws:iam::<aws_account_number>:user/<aws_username>'
+? Create the 'ManagedOpenShift-Installer-Role' role? Yes <5>
+I: Created role 'ManagedOpenShift-Installer-Role' with ARN 'arn:aws:iam::<aws_account_number>:role/ManagedOpenShift-Installer-Role'
+? Create the 'ManagedOpenShift-ControlPlane-Role' role? Yes <5>
+I: Created role 'ManagedOpenShift-ControlPlane-Role' with ARN 'arn:aws:iam::<aws_account_number>:role/ManagedOpenShift-ControlPlane-Role'
+? Create the 'ManagedOpenShift-Worker-Role' role? Yes <5>
+I: Created role 'ManagedOpenShift-Worker-Role' with ARN 'arn:aws:iam::<aws_account_number>:role/ManagedOpenShift-Worker-Role'
+? Create the 'ManagedOpenShift-Support-Role' role? Yes <5>
+I: Created role 'ManagedOpenShift-Support-Role' with ARN 'arn:aws:iam::<aws_account_number>:role/ManagedOpenShift-Support-Role'
+I: To create a cluster with these roles, run the following command:
+rosa create cluster --sts
+----
+<1> Specify the prefix to include in the {cluster-manager} IAM role name. The default is `ManagedOpenShift`.
++
+[IMPORTANT]
+====
+You must specify an account-wide role prefix that is unique across your AWS account, even if you use a custom ARN path for your account roles. The custom paths provide only a logical separation for your IAM roles within your account.
```

Comment:
I don't understand what "only" means in this sentence

Reply:
Thank you for highlighting this. The custom paths provide a way to logically categorise each set of account wide roles. For example, you could have a set in `/custom/path/dev/` and a set in `/custom/path/prod/`. But, those two sets still need to have unique names. You could not have two sets of account-wide roles with the same prefix name in two different custom paths in the same way that you could have two identical file names in separate Linux directories. This is what I meant by only a logical separation, rather than actually separating the roles and policies as if they were in separate directories.

On review of your point, I decided it's best just to remove the second sentence from the line. The requirement is clear from the first sentence alone.